### PR TITLE
Preferred report language

### PR DESCRIPTION
**Pull request by EvsenevDev**
- Added "Preferred Report Language" feature
- Don't apply paid features if user don't have valid subscription

### CODE REVIEW

File: SmartReceipts/Persistence/WBPreferences.swift [hunk 1]
```diff
@@ -6,21 +6,56 @@
...
+    
+    static func languageBy(identifier: String) -> LanguageAlias? {
+        return WBPreferences.languages.first { identifier == $0.identifier }
+    }
+    
+    static func languageBy(name: String) -> LanguageAlias? {
+        return WBPreferences.languages.first { name == $0.name }
+    }
+    
+    @objc class func loclized(key: String, comment: String = "") -> String {
```
Comment (wbaumann) [inline SmartReceipts/Persistence/WBPreferences.swift]: Could you rename this to `localized` from `loclized`. There's a missing `a`

**Comment by wbaumann** [comment]
Looks good. One minor request first to fix a typo

File: SmartReceipts/Persistence/WBPreferences.swift (hunk 1 above)
Comment (EvsenevDev) [reply SmartReceipts/Persistence/WBPreferences.swift]: yes, thanks

**Comment by wbaumann** [comment]
@EvsenevDev Thanks :)

**Pull request merged**